seed/seedwriter/manifest_test.go

```diff
@@ -55,8 +55,8 @@ func (s *manifestSuite) writeSeedManifest(c *C, contents string) string {
 	return manifestFile
 }
 
```

Comment:
Did a scan of the file (`grep -rin "SeedManifest"`), and picked up the following for consideration: 

```
manifest_test.go:51:func (s *manifestSuite) writeSeedManifest(c *C, contents string) string {
manifest_test.go:63:func (s *manifestSuite) TestReadSeedManifestFullHappy(c *C) {
manifest_test.go:66:	manifestFile := s.writeSeedManifest(c, `# test line should not match
manifest_test.go:87:func (s *manifestSuite) TestReadSeedManifestParseFails(c *C) {
manifest_test.go:105:		manifestFile := s.writeSeedManifest(c, t.contents)
manifest_test.go:111:func (s *manifestSuite) TestReadSeedManifestNoFile(c *C) {
manifest_test.go:118:func (s *manifestSuite) testWriteSeedManifest(c *C, revisions map[string]*seedwriter.ManifestSnapRevision, vss map[string]*seedwriter.ManifestValidationSet) string {
manifest_test.go:126:func (s *manifestSuite) TestWriteSeedManifestNoFile(c *C) {
manifest_test.go:127:	filePath := s.testWriteSeedManifest(c, nil, nil)
manifest_test.go:131:func (s *manifestSuite) TestWriteSeedManifest(c *C) {
manifest_test.go:132:	filePath := s.testWriteSeedManifest(c,
manifest_test.go:160:func (s *manifestSuite) TestSeedManifestSetAllowedSnapRevisionInvalidRevision(c *C) {
manifest_test.go:166:func (s *manifestSuite) TestSeedManifestSetAllowedSnapRevision(c *C) {
manifest_test.go:175:func (s *manifestSuite) TestSeedManifestSetAllowedSnapRevisionTwice(c *C) {
manifest_test.go:188:func (s *manifestSuite) TestSeedManifestMarkSnapRevisionSeededWithAllowedHappy(c *C) {
manifest_test.go:204:func (s *manifestSuite) TestSeedManifestMarkSnapRevisionSeededNoMatchingAllowed(c *C) {
manifest_test.go:217:func (s *manifestSuite) TestSeedManifestMarkSnapRevisionSeededTwice(c *C) {
manifest_test.go:225:func (s *manifestSuite) TestSeedManifestMarkSnapRevisionSeededWrongRevision(c *C) {
manifest_test.go:233:func (s *manifestSuite) TestSeedManifestSetAllowedValidationSet(c *C) {
manifest_test.go:247:func (s *manifestSuite) TestSeedManifestSetAllowedValidationSetTwice(c *C) {
manifest_test.go:260:func (s *manifestSuite) TestSeedManifestSetAllowedValidationSetInvalidSequence(c *C) {
manifest_test.go:296:func (s *manifestSuite) TestSeedManifestMarkValidationSetSeededUsedHappy(c *C) {
manifest_test.go:364:func (s *manifestSuite) TestSeedManifestMarkValidationSetWeirdCases(c *C) {
manifest_test.go:383:func (s *manifestSuite) TestSeedManifestMarkValidationSetSeededUsedTwice(c *C) {
manifest_test.go:393:func (s *manifestSuite) TestSeedManifestMarkValidationSetSeededWrongSequence(c *C) {
manifest_test.go:403:func (s *manifestSuite) TestSeedManifestMarkValidationSetSeededWrongPinned(c *C) {
```

Reply:
all of these are fixed in https://github.com/snapcore/snapd/pull/12732/commits/14e31c67df011fffa68e408d0c7730c373cb10f8